client-src/index.js

```diff
@@ -58,6 +58,30 @@ if (parsedResourceQuery["live-reload"] === "true") {
   log.info("Live Reloading enabled.");
 }
 
+if (parsedResourceQuery.progress === "true") {
+  options.progress = true;
+
+  log.info("Progress reporting enabled.");
+}
+
+if (parsedResourceQuery.overlay) {
+  try {
+    options.overlay = JSON.parse(parsedResourceQuery.overlay);
+    log.info("Error overlay enabled.");
+  } catch (e) {
+    log.error("Error parsing overlay options from resource query:", e);
+  }
+
+  // Fill in default "true" params for partially-specified objects.
+  if (typeof options.overlay === "object") {
```

Comment:
according to the [docs](https://webpack.js.org/configuration/dev-server/#overlay) the defaults should be true (though i don't see where this is implemented when using the overlay object options in the webpack config)